### PR TITLE
Clean up recent patients

### PR DESCRIPTION
**Pull request by saket**
I was trying to fix a bug in recent patients, but I'll need admin access to sandbox before I can continue my investigation. This PR contains changes I made in the meanwhile to clean up some code.

https://www.pivotaltracker.com/n/projects/2184102/stories/164720063

### CODE REVIEW

**Comment by saket** [comment]
@vinaysshenoy the last commit's description contains screenshots of before/after the commit is applied.

File: app/src/main/res/layout/recent_patients.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     style="@style/Clinic.V2.TextAppearance.Subtitle1Center.Grey1"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginBottom="@dimen/spacing_96"
+    android:layout_marginBottom="104dp"
```
Comment (vinaysshenoy) [inline app/src/main/res/layout/recent_patients.xml]: I die a little inside every time I see a hardcoded value. 😢

**Comment by vinaysshenoy** [comment]
@saket Looks good to me, but one of the tests (`verify_recent_patients_are_retrieved_as_expected`) seems to be failing. Looks like flakiness caused by generating timestamps for test data since the only thing that's unexpected is the order of the results. Take a look?

File: app/src/main/res/layout/recent_patients.xml (hunk 1 above)
Comment (saket) [reply app/src/main/res/layout/recent_patients.xml]: Ohh but why? This value isn't shared with any screen. What do you suggest?

Comment (vinaysshenoy) [reply app/src/main/res/layout/recent_patients.xml]: Oh, I just meant in general. :P

**Comment by saket** [comment]
@rakshakhegde maybe you should also take a look at the changes, because you originally wrote most of the changed code.

**Pull request merged**